### ISSUE TITLE
Why are the features extracted by VGG not used as loss terms

### ISSUE THREAD

**Issue by shi991027** [jinyeying/night-enhancement#28]
Thank you very much for your work, but I found that in the training code for the Light-Effects Suppression section,ENHANCENET.py, it does not use VGG to extract features from the original and generated images and use this as a loss term, which is mentioned in the paper, why is this?

**Comment by jinyeying** [comment]
> Thank you very much for your work, but I found that in the training code for the Light-Effects Suppression section,ENHANCENET.py, it does not use VGG to extract features from the original and generated images and use this as a loss term, which is mentioned in the paper, why is this?

refer to PerceptualLossVgg16ExDark(nn.Module) in demo.py
